1a_single_site/slurm_distributed_workflow/README.md

```diff
@@ -0,0 +1,225 @@
+```sh
+git clone https://github.com/ccmmf/workflows.git
+```
+
+The environment tarball and data artifacts have been hosted by NCSA, and can be obtained via the S3 protocol from:
+```sh
+s3.garage.ccmmf.ncsa.cloud
+```
+
+Typically, you will be able to leverage the AWS CLI toolset to access these resources.
+
+Once you enter the needed Access key and Secret Access Key, e.g.:
+```sh
+AWS Access Key ID [None]: GK8bb0d9c6b355c9a25b0b67fa
+AWS Secret Access Key [None]: <-- secret key to be passed via other method -->
+Default region name [None]: garage
+Default output format [None]: 
+```
+
+You can then identify the CARB resources by identifying the correct endpoint via the 'endpoint-url' parameter. 
+For example this command will recursively list the contents of the carb bucket:
+```sh
+aws s3 ls --recursive --endpoint-url https://s3.garage.ccmmf.ncsa.cloud s3://carb
+```
+
+
+## Head-Node installation: Conda & Conda Environment {#headnodeinstallation}
+If you do not already have Conda installed, a good alternative for a local user install is miniconda. Install miniconda following [these instructions](https://www.anaconda.com/docs/getting-started/miniconda/install#linux)
+
+The below commands assume you keep your conda environments in the standard location: ```{USER}/.conda/envs```
+
+The pre-packaged headnode environment can be obtained from the S3 data host with this command:
+```sh
+aws s3 cp --endpoint-url https://s3.garage.ccmmf.ncsa.cloud \
+  s3://carb/environments/PEcAn-head.tar.gz ./
+
+```
+
+If you have not used conda before, it is suggested you unpack this environment into the standard location:
+```sh
+mkdir ~/.conda
+```
+```sh
+mkdir ~/.conda/envs
+```
+```sh
+mkdir ~/.conda/envs/PEcAn-head
+```
+```sh
+tar -xzf PEcAn-head.tar.gz -C ~/.conda/envs/PEcAn-head
+```
+```sh
+source ~/.conda/envs/PEcAn-head/bin/activate
+```
+```sh
+conda-unpack
+```
+At this point, the conda environment is unpacked, and the 'conda-unpack' command has adjusted the paths within the environment to match your local filesystem. You should be able to interrogate the conda environment's installation of R to confirm this:
+
+```sh
+Rscript -e '.libPaths()'
+```
+This should yield output that points to the R-library location within the unpacked conda environment.
+```sh
+[1] "/home/hdpriest/.conda/envs/PEcAn-head/lib/R/library"
+# the above path will reflect local file system home and user specifics
+```
+
+In addition, you should be able to access the portions of the PEcAn software stack that are needed on the headnode of the cluster:
+```sh
+Rscript -e 'library("PEcAn.workflow")'
+```
+or
+```sh
+Rscript -e 'library("PEcAn.remote")'
+```
+You __will__ need to have this environment activated when executing work in a Slurm-scheduled manner, as the job submissions to the Slurm schedule are enabled via PEcAn methods.
+
+Typically, this environment can be activated via:
+```sh
+conda activate PEcAn-head
+```
+
+## Distributed PEcAn Workflows {#distributedpecan}
```

Comment:
Somewhere here -- maybe in this section but I haven't thought carefully -- it would be good to summarize what differs between the slurm and non-slurm workflows. 

As we discussed live, I think it's a worthy eventual goal to reconverge these, hopefully all the way into a single set of scripts that work right (potentially with a configuration step) for both serial and Slurm-enabled runs.

Reply:
Suggested an additional blurb below